services/venice-controller/src/main/java/com/linkedin/venice/controller/Admin.java

```diff
@@ -436,7 +436,7 @@ String getNativeReplicationSourceFabric(
 
   TopicManager getTopicManager();
 
-  TopicManager getTopicManager(Pair<String, String> kafkaBootstrapServersAndZkAddress);
+  TopicManager getTopicManager(String pubSubServerAddress);
```

Comment:
Good to change this param name... although I wonder if it will be sufficient to have just one param, or if we'll need to pass a property bag to support a greater variety of pub sub systems. We can cross that bridge when we get there.

Reply:
I think we've been passing properties to factories and just broker addresses to override for topicManager(). But Yeah we can evolve it to use generic config arg.